bedrock/firefox/templates/firefox/new/trailhead/thanks-a.html

```diff
@@ -0,0 +1,83 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros-protocol.html" import hero, picto_card with context %}
+
+{% extends "firefox/new/trailhead/base.html" %}
+
+{% block extrahead %}
+  {{ super() }}
```

Comment:
If you extend from `thanks.html`, then also make this change:

```suggestion
  {{ css_bundle('protocol-core') }}
```